hokusai/staging.yml

```diff
@@ -113,3 +113,37 @@ spec:
     component: web
   sessionAffinity: None
   type: LoadBalancer
+---
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: kaws-update-sailthu
```

Comment:
General question — do we actually want to run this cron job in _staging_ as well? Do we even have separate production and staging environments in Sailthru? Or would we end up clobbering our single Sailthru instance with our staging Kaws data?

Reply:
Staging and Prod have different sailthru credentials in their environments, so I _think_ this is fine - plus it lets us confirm on staging that it's working.

Reply:
👌